images/ova/rhel-79.yaml

```diff
@@ -13,7 +13,7 @@ packer:
   insecure_connection: "false"
   network: ""
   resource_pool: ""
-  template: "base-rhel-7"
+  template: "base-rhel-7-sortega" 
```

Comment:
What is "sortega"? This is customer facing file, is this d2iq specific?

Reply:
This is the name of our base image. Our ones named `base-rhel-7` were not configured correctly.

Reply:
Thats what I figured, but is "sortega" meaningful outside of D2IQ and our template? This is changing an external file used by our customers. I think it would be unexpected for them that an argument changed.

Reply:
Thanks. this is a good point. I can rename the `base-rhel-7-sortega` to `base-rhel-7` in vsphere.